app/api/evaluation/route.ts

```diff
@@ -220,16 +238,17 @@ export async function GET(req: NextRequest) {
           },
         });
       } else {
+        const traceService = await new TraceService();
+        const tempSpanId = await traceService.GetSpanById(spanId, projectId);
```

Comment:
This is breaking the SDK! SDK does not pass projectId with sendUserFeedback so this API call fails in the SDK when it tries to GET to check if there is an existing eval or not to decide whether to make a POST or a PUT.

Reply:
And we cannot update SDK to pass project ID cos it adds friction